apps/rs/README.md

```diff
@@ -0,0 +1,39 @@
+Halide Renderscript "blur" and "copy" samples
+===
+
+These samples demonstrates two supported modes of operation for Renderscript codegen:
+ - vectorized operations over interleavead RGBA images;
+ - single-element operations over planar images.
+
+Prerequisites
+---
+
+You will need to have Android NDK in the path.
+This app only runs on arm-based Android devices.
```

Comment:
One other comment here is that we would want to look at generating Java wrapper around Renderscript Halide filters. Current solution relies on native (arm or x86) code to wrap Renderscript, which is not as portable as Java.